src/app/story/directives/player.component.ts

```diff
@@ -1,122 +1,175 @@
-        are placeholders until this episode has been published.
-      </p>
-        <iframe
-                [src]='this.safeUrl'
-                frameborder="0"
-                height="200"
-                width ="100%">
-        </iframe>
-
-        <p>You can embed this player on your own site by including the following <code>iframe</code> tag.</p>
-
-        <div class="embed-code">
-          <input type="text" [value]="iframeHtml(200, 650)" #shareEmbedSmall readonly>
-          <button (click)="copy(small)" data-copytarget="#shareEmbedSmall" #small>Copy</button>
-        </div>
-
-    </div>
-  `
+  templateUrl: 'player.component.html'
 })
 
-export class PlayerComponent implements OnDestroy {
-  @ViewChild('shareEmbedSmall') private inputEl: ElementRef;
+export class PlayerComponent implements OnDestroy, DoCheck {
+
+  private tabSub: Subscription;
+  private story: StoryModel;
+
+  private feedUrl: string;
+  private episodeGuid: string;
   private title: string;
   private subtitle: string;
   private audioUrl: string;
   private imageUrl: string;
   private subscriptionUrl: string;
 
-  story: StoryModel;
-  tabSub: Subscription;
-
-  constructor(
-    tab: TabService,
-    private sanitizer: DomSanitizer
-  ) {
-    this.tabSub = tab.model.subscribe((s: StoryModel) => {
-      this.setFields(s);
+  shouldUseFeeder: boolean;
+  previewingUnpublished: boolean;
+  loadError: string;
+
+  previewUrl: string;
+  previewSafe: SafeResourceUrl;
+  copyUrl: string;
+  copyIframe: string;
+
+  constructor(tab: TabService, private sanitizer: DomSanitizer) {
+    this.tabSub = tab.model.subscribe((story: StoryModel) => {
+      this.story = story;
+      story.getSeriesDistribution('podcast').subscribe(distDoc => {
+        this.shouldUseFeeder = distDoc ? true : false;
+        this.fromStory(story);
+        if (distDoc) {
+          this.fromFeeder(story, new DistributionModel(story.parent, distDoc));
+        }
+      });
     });
   }
 
-  setFields(story: StoryModel) {
-    this.story = story;
-    this.title = story.title || 'Placeholder title';
-    this.subtitle = story.shortDescription || 'Placeholder subtitle';
-    this.subscriptionUrl = ''; // currently not in place
-
-    if (this.story.images.length > 0) {
-      let img = this.story.images.find(img => !img.isDestroy);
-      if (img) { this.imageUrl = img.enclosureHref; }
-    }
+  ngOnDestroy(): any {
+    this.tabSub.unsubscribe();
+  }
 
-    if (this.story.versions.length > 0) {
-      let firstAudio = this.story.versions[0].files.find(file => !file.isDestroy);
-      if (firstAudio) { this.audioUrl = firstAudio.enclosureHref; }
+  ngDoCheck() {
+    let hasCmsParams = this.title && this.subtitle && this.audioUrl && this.imageUrl && this.subscriptionUrl;
+    let hasFeederParams = this.feedUrl && this.episodeGuid;
+    if (this.shouldUseFeeder === false && hasCmsParams) {
+      this.previewingUnpublished = false;
+      this.setCmsLoadError();
```

Comment:
Is it right that you only see the message about default audio + image being in the preview if this is a standalone story? Since cover image isn't required in Basic tab, very possible that a user creating a story in a podcast will get here without an imageUrl, in which case the unpublished preview they see will have the default image. Fine to leave that as is? I've argued both sides to myself. 
In any case, I'll approve as is and you can do as you see fit!